tools/image_digest_.py

```diff
@@ -0,0 +1,110 @@
+# Copyright 2018 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This package calculates digest of an image.
+
+The format this tool *expects* to deal with is proprietary.
+"""
+
+from __future__ import absolute_import
+
+from __future__ import print_function
+
+import argparse
+import logging
+import sys
+
+from containerregistry.client.v2_2 import docker_image as v2_2_image
+from containerregistry.client.v2_2 import oci_compat
+from containerregistry.tools import logging_setup
+
+from six.moves import zip  # pylint: disable=redefined-builtin
+
+
+parser = argparse.ArgumentParser(
+    description='Calculate digest for a container image.')
+
+parser.add_argument(
```

Comment:
Please specify types and whether is required for these flag values.

Reply:
They are all strings, except for the `--oci` which is boolean and `action='store_true'` ensures it is either true or false. Is that ok or I need to explicitly put all types there? Please note that I've just copied [fast_pusher_.py](https://github.com/atlassian/containerregistry/blob/0492daa2652d500ea618bebe66cdf2e3a80d030c/tools/fast_pusher_.py) and removed what is not needed, following the style as close as possible. It does not have types or `required` markers.

Reply:
It's up to you at this point. I'm happy with what you have now. You don't _have_ to follow fast_pusher_.py code style.